src/transformers/utils/import_utils.py

```diff
@@ -246,6 +246,16 @@
 except importlib_metadata.PackageNotFoundError:
     _librosa_available = False
 
+ccl_version = "N/A"
+_is_ccl_available = (
+    importlib.util.find_spec("torch_ccl") is not None
+    or importlib.util.find_spec("oneccl_bindings_for_pytorch") is not None
+)
+try:
+    ccl_version = importlib_metadata.version("oneccl_bind_pt")
```

Comment:
Very confused by this line since the name is different from the two packages. Is there an alias in both packages to this?

Reply:
the package name is oneccl_bind_pt. and from 1.12, the module name has changed from torch_ccl to oneccl_bindings_for_pytorch

Reply:
It is not two packages, 'oneccl_bindings_for_pytorch' is the module name of the 'oneccl_bind_pt' package. oneccl_bind_pt is just a  abbreviation of oneccl_bindings_for_pytorch. can you help to add comment in the code about this? @sywangyi

Reply:
@sgugger ,have uploaded document